peekingduck/pipeline/nodes/input/live.py

```diff
@@ -47,7 +48,9 @@ def run(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
                 img = resize_image(img,
                                    self.resize_info['width'],
                                    self.resize_info['height'])
-            outputs = {self.outputs[0]: img}
+            outputs = {self.outputs[0]: img, "fps":self.fps_saved_output_video,"filename":"webcamfeed.mp4"}
```

Comment:
It may be good to lift the filename as a config variable in live.yml .

Reply:
i added the filename as a config variable in the latest commit

Reply:
Thanks